src/components/table/JobTable.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import clsx from 'clsx';
+import { WithStyles, withStyles } from '@mui/styles';
+import moment from 'moment';
+import { JobModel } from 'generated/tdr';
+import { OrderDirectionOptions, TableColumnType } from 'reducers/query';
+import { CustomTheme } from '@mui/material/styles';
+
+import LightTable from './LightTable';
+
+const styles = (theme: CustomTheme) => ({
+  textWrapper: {
+    ...theme.mixins.ellipsis,
+  },
+  statusContainer: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  statusIcon: {
+    fontSize: '1rem',
+    marginRight: 10,
+  },
+  statusIconSuccess: {
+    color: theme.palette.success.light,
+  },
+  statusIconFailed: {
+    color: theme.palette.error.light,
+  },
+});
+
+interface IProps extends WithStyles<typeof styles> {
+  jobs: Array<JobModel>;
+  jobRoleMaps: { [key: string]: Array<string> };
+  jobsCount?: number;
+  filteredJobsCount?: number;
+  handleFilterJobs?: (
+    rowsPerPage: number,
+    rowsForCurrentPage: number,
+    orderProperty: string,
+    orderDirection: OrderDirectionOptions,
+    searchString: string,
+    refreshCnt: number,
+  ) => void;
+  handleMakeSteward?: (jobId: string) => void;
+  loading: boolean;
+  searchString: string;
+  refreshCnt: number;
+}
+
+const JobTable = withStyles(styles)(
+  ({ classes, jobs, handleFilterJobs, loading, searchString, refreshCnt }: IProps) => {
+    const statusMap: any = {
+      succeeded: { icon: `fa fa-circle-check ${classes.statusIconSuccess}`, label: 'Completed' },
+      running: { icon: `fa fa-rotate fa-spin ${classes.statusIconSuccess}`, label: 'In Progress' },
+      failed: { icon: `fa fa-circle-xmark ${classes.statusIconFailed}`, label: 'Failed' },
+    };
+
+    const columns: Array<TableColumnType> = [
+      {
+        label: 'Job ID',
+        name: 'id',
+        allowSort: true,
```

Comment:
We can't sort on any column other than submitted time (and I'm not sure we even want to allow sorting on that).  Let's set allowSort to false on all of these